requirements.txt

```diff
@@ -0,0 +1,3 @@
+Django==1.11
+pkg-resources==0.0.0
```

Comment:
I think we can safely remove this line. This package seems to be a bug when creating requirements.txt from within virtual environments. Refer to [this](https://stackoverflow.com/a/40167445/8115428).